crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -847,6 +847,7 @@ impl StickyData for SlidingSyncStickyParameters {
 /// TODO remove this workaround as soon as support of the `limited` flag is
 /// properly implemented in the open-source proxy: https://github.com/matrix-org/sliding-sync/issues/197
 // NOTE: SS proxy workaround.
+#[instrument(level = "trace", skip_all)]
```

Comment:
The span isn't usually printed when no event happens within, so I don't think overriding the default level here is really necessary. That's not to say it's bad though.